src/test/java/com/cook4me/controller/OfferManagerTest.java

```diff
@@ -0,0 +1,90 @@
+package com.cook4me.controller;
+
+import com.cook4me.model.Offer;
+import org.junit.*;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+
+/**
+ * Created by Gaba on 2016-12-08.
+ */
+
+public class OfferManagerTest {
+    private Offer offer;
+    private OfferManager offerManager;
+    private Offer nullOffer;
+    private Offer offerWithNullFields;
+
+    @Before
+    public void setUp(){
+        offerManager = new OfferManager();
+        offer = new Offer("bigos", "Krakow", "20.12.2016", "yummy stuff", "880880880");
+        nullOffer = null;
+        offerWithNullFields = new Offer(null, null, null, null, null);
+
+    }
+
+    @Test
+    public void idShouldBeAddedToOffer(){
+        offerManager.store(offer);
+        int actualOfferListSize = offerManager.getOffers().size();
+        long offersId = offer.getId();
+        Offer actualOffer = offerManager.getOffer(offersId);
+        List actualListOfOffers = offerManager.getOffers();
+
+        assertEquals("The size of list of offers should be 1.", 1, actualOfferListSize);
+        assertEquals("Input offer should be returned.", offer, actualOffer);
+        assertEquals("Name of returned offer should be <<bigos>>.", "bigos", actualOffer.getName());
+        assertEquals("First element of list of offers should be input offer.", offer, actualListOfOffers.get(0));
+        assertNull("Accessing offer with id that is not in the list should return null.", offerManager.getOffer(offersId + 24));
```

Comment:
Should offerManager return null? Or should it throw an exception? What do you think? @gaabral @LordVader14